webxrlayers-1.bs

```diff
@@ -1034,6 +1056,116 @@ interface XRMediaBinding {
 
 </pre>
 
+Each {{XRMediaBinding}} has an associated <dfn dfn-for="XRMediaBinding">session</dfn>, which is the
+{{XRSession}} it was created with.
+
+NOTE: It is possible to create more than one {{XRMediaBinding}}. The lifetime of a layer is not tied
+to the lifetime of the {{XRMediaBinding}} that created it.
+
+Each layer created through {{XRMediaBinding}} has an internal {{HTMLVideoElement}} [=XRCompositionLayer/media=].
+If the layer is part of the session's {{XRSession/renderState}}, it will display the current frame of the video. The
+layer is update at the native framerate of the [=/XR device=] or the video, whichever is less.
+
+NOTE: only the video frames will be displayed in the layer. Video controls should be implemented by the author and must
```

Comment:
unsure if we should have this, but it can be discussed in another issue

Reply:
maybe have an ISSUE line for this

Reply:
done